_posts/2021-12-15-the-foreman-ecosystem-and-cve-2021-44228.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: Foreman and CVE-2021-44228
+date: 2021-12-15 13:06:10
+author: Melanie & Nikos
+tags:
+- foreman
+---
+
+More information for anyone following CVE-2021-44228, who wants to understand whether Foreman and the common projects it interacts with are impacted.
```

Comment:
Should we add a tl;dr? Something like "Foreman is unaffected by CVE-2021-44228" can help ease users mind before even reading it and they know this is only a more in-depth explanation.